WcaOnRails/.rubocop_todo.yml

```diff
@@ -234,6 +234,7 @@ Style/ClassAndModuleChildren:
     - 'app/controllers/accounts/registrations_controller.rb'
     - 'app/controllers/api/v0/api_controller.rb'
     - 'app/controllers/api/v0/competitions_controller.rb'
+    - 'app/controllers/api/v0/records_controller.rb'
```

Comment:
Since you asked, I'd probably not have a rule for this or most things Rubocop enforces at all.